views/js/GoogleAnalyticActionLib.js

```diff
@@ -116,7 +117,7 @@ var GoogleAnalyticEnhancedECommerce = {
 		ga('ec:setAction', 'refund', {
 			'id': Order.Id, // Transaction ID is required for partial refund.
 		});
-		//ga('send', 'pageview');
+		ga('send', 'event','Refund','refund',{'nonInteraction':1});
```

Comment:
Regarding the [Google Analytics documentation](https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce#measuring-refunds):
```js
ga('send', 'event', 'Ecommerce', 'Refund', {'nonInteraction': 1});
```